tests/test_pytest_rerunfailures.py

```diff
@@ -3,7 +3,6 @@
 from unittest import mock
 
 import pytest
-
```

Comment:
I think we need to add this to `pyproject.toml` to prevent this:

[tool.ruff.lint.isort]
known-first-party = ["pytest_rerunfailures"]

Reply:
done